_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,329 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that there are instruments to pause, stop or hide the auto-updating of text content.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML element][] that has a [visible][] [text node][] as a [descendant][] in the [flat tree][] if:
+
+- **changed:** the `innerText` property of the [element][html element] changes multiple times within a 10 minute time span where there is no [user interaction][]; and
```

Comment:
- "update" does not encompass all the possible changes that might affect the innerText property. What if I remove the innerText? Would you say it was updated? In the very least, updated would be ambiguous then. But it changed.

- "based on preset interval" could imply that the updates have a determined frequency (because there is a preset interval). What if the updates are pushed by a server when a specific event occurs (which could be random, for all we know)? That is why multiple times was used.

- There is a note below explaining the "10 minutes". Don't you think it is enough? If we remove the arbitrary time span, it means that we would never be able to test the rule because one could never say that an update would not happen in the future (unless you inspected the scripts in all pages you are evaluating).

Reply:
@adilsofficial let me begin by telling you that what you are suggesting is mostly how this rule looked like when I started writing it :-) Then, as you can see, it moved away from that after several rounds of reviews

- The "update" word. I think it was Wilco who raised this one. "Update" can be interpreted as you are interpreting (which was also my initial interpretation), but can also be interpreted as referring only to changes in the content of something (similar to what an update does in a database). If the "update" involves removing an element and re-creating the element with new content via a script, it should still be considered an update for this rule. But some people will not interpret that as an update (they will see it has a delete followed by a create, using the database terminology). That is why we replaced "update" by "change".

- The "preset interval". This one was raised by Jean-Yves, I believe. First, how do you determine that there is a preset interval (remember, this is applicability, so it needs to be objective)? Do you require the evaluator to read every possible script (including potentially server-side scripts) that is used on the web page to find that out? And what if the because the processor has a higher load, the processing of the event that updates the content takes an extra millisecond? Or the network is slower, and the update reaches the page one second after what it should? Then your updated frequency changes, and you cannot define a "preset interval". That is why, instead of trying to come up with an objective solution for this problem, we went with the "changes multiple times" solution.

You can find more detailed arguments for these issues in the history of comments for this PR :-)

Let us know if using "multiple times" in this rule is something that will prevent you from approving it (and if so, it would be great if you can come up with a way to solve the issue of the frequency/preset interval)